frontend/packages/kubevirt-plugin/src/components/create-vm/use-vm-templates-resources.tsx

```diff
@@ -0,0 +1,78 @@
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { PersistentVolumeClaimKind, PodKind, TemplateKind } from '@console/internal/module/k8s';
+import { PersistentVolumeClaimModel, PodModel, TemplateModel } from '@console/internal/models';
+import { DataVolumeModel } from '../../models';
+import { V1alpha1DataVolume } from '../../types/vm/disk/V1alpha1DataVolume';
+import {
+  TEMPLATE_TYPE_BASE,
+  TEMPLATE_TYPE_VM,
+  TEMPLATE_TYPE_LABEL,
+  CDI_APP_LABEL,
+} from '../../constants';
+import { useBaseImages } from '../../hooks/use-base-images';
+
+export const useVmTemplatesResources = (namespace): useVmTemplatesResourcesValues => {
+  const [userTemplates, utLoaded, utError] = useK8sWatchResource<TemplateKind[]>({
+    kind: TemplateModel.kind,
+    namespace,
+    selector: {
+      matchLabels: { [TEMPLATE_TYPE_BASE]: TEMPLATE_TYPE_VM },
```

Comment:
this is why it wont load any user templates :) 

```suggestion
      matchLabels: { [TEMPLATE_TYPE_LABEL]: TEMPLATE_TYPE_VM },
```